src/components/cards/challenge/Challenge.tsx

```diff
@@ -25,81 +24,69 @@ interface ChallengeCardProps {
+  return (
+    <div className="w-full justify-between- flex flex-col sm:flex-row  md:flex-col lg:flex-row  border-solid border border-gray-200 bg-gray-50 rounded-3xl mb-5 group text-gray-700">
+      <div className="border-solid border-b border-gray-300 bg-white rounded-3xl sm:p-8 sm:pb-6 p-6 lg:w-2/3">
+        <div className="flex flex-col mb-1">
+          <div className="lg:pr-20 lg:w-3/5-">
             <div className="text-gray-400 lg:w-2/3 text-sm font-normal mb-6 lg:mb-0">
               {expiresAt && (
                 <>
-                  <span>Deadline</span>
+                  <span>{t("communities.overview.challenge.deadline")}</span>
                   <span className="font-medium pl-1">{expiresAt}</span>
                 </>
               )}
             </div>
-
-            <Link href={link}>
-              <ArrowButton communityStyles={true} variant="outline-primary">
-                {isCourseEnd ? "Take the challenge" : "See the challenge"}
-              </ArrowButton>
-            </Link>
+            <div className="text-lg text-gray-900 font-medium leading-normal mb-6">{data.name}</div>
+            <Badges challenge={data} className="" />
+            <div className="text-sm pb-2 text-gray-700">{data.description}</div>
+          </div>
+          <div className="divide-y-2 divide-gray-200 divide-dotted flex flex-col mt-3">
+            <p className="pb-6">
+              {learningModulesCount ? `${learningModulesCount}  learning ${learningModulesCount === 1 ? "module" : "modules"} included` : "No learning modules included"}
+            </p>
+            <div className="lg:flex lg:flex-row flex-col justify-between pt-6 items-center">
+              <Link href={link}>
+                <ArrowButton communityStyles={true} variant="outline-primary">
+                  {isCourseEnd ? t("communites.overview.challenge.take.challenge") : t("communities.overview.challenge.see.challenge")}
+                </ArrowButton>
+              </Link>
+            </div>
           </div>
         </div>
       </div>
-
-      {(data.courses?.length > 0 || data.learningModules?.length > 0) && !isCourseEnd && (
-        <div className="sm:px-8 sm:pt-6 sm:pb-9 w-full p-6 rounded-3xl text-sm">
-          <div className="mb-3 text-gray-400 font-semibold uppercase text-xxs">related content</div>
-          {data.courses?.map((course) => (
-            <RelatedContent key={course.id} content={course} />
-          ))}
-          {data.learningModules?.map((learningModule) => (
-            <RelatedContent key={learningModule.id} content={learningModule} />
+      <div className="p-6">
+        <div className={"mx-auto relative rounded-full mb-5 w-[147px] h-[184px]"}>
+          <Image src={data?.certificateData?.icon} alt="achievement" fill />
+        </div>
+        <div className="">
+          <h1 className="font-bold text-gray-400 text-xs">{t("communities.overview.challenge.unlock.certificate").toUpperCase()}</h1>
```

Comment:
I would recommand to use uppercase from CSS, 
```suggestion
          <h1 className="font-bold text-gray-400 text-xs uppercase">{t("communities.overview.challenge.unlock.certificate")}</h1>
```